### PR TITLE
Update fcm library and improved app stability.

### PR DESCRIPTION
**Pull request by lyw0149**
1. Update react-native-fcm module :  2.3.2 -> 2.5.3.
2. Use InteractionManager and delete setTimeout func.

### CODE REVIEW

File: ios/Bridgeme.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -13,6 +13,7 @@
 		00C302E91ABCBA2D00DB3ED1 /* libRCTNetwork.a in Frameworks */ = {isa = PBXBuildFile; fileRef = 00C302DC1ABCB9D200DB3ED1 /* libRCTNetwork.a */; settings = {ATTRIBUTES = (Required, ); }; };
 		00C302EA1ABCBA2D00DB3ED1 /* libRCTVibration.a in Frameworks */ = {isa = PBXBuildFile; fileRef = 00C302E41ABCB9EE00DB3ED1 /* libRCTVibration.a */; settings = {ATTRIBUTES = (Required, ); }; };
 		00E356F31AD99517003FC87E /* yodaTests.m in Sources */ = {isa = PBXBuildFile; fileRef = 00E356F21AD99517003FC87E /* yodaTests.m */; };
+		01E4626FFA314B2907F86752 /* Pods_Bridgeme.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 8413B23E955A24C014BF81B2 /* Pods_Bridgeme.framework */; };
```
Comment (ace0625) [inline ios/Bridgeme.xcodeproj/project.pbxproj]: In this PR, I don't think we need to update project file.

File: src/components/Chat/Row.js [hunk 2]
```diff
@@ -118,7 +122,7 @@ export default class Row extends Component {
 
 const styles = StyleSheet.create({
   row: {
-    flex: 1,
+    height: 75,
```
Comment (ace0625) [inline src/components/Chat/Row.js:125]: Could you evaluate the row size to keep the same proportion in all devices?

File: src/components/Main.js [hunk 3]
```diff
@@ -131,36 +135,40 @@ class Main extends Component {
   }
 
   onMainMessageReceived(channel, userMessage) {
-    Vibration.vibrate();
     FcmUtil.presentLocalChatNotification(userMessage);
   }
 
   onNotificationReceived(notif) {
     if (notif.opened_from_tray) {
```
Comment (ace0625) [inline src/components/Main.js:142]: Could you change opened_from_tray variable to camel case?

**Review by ace0625 — COMMENTED** [review]
Overall, Good update for chat. Luke!

File: src/components/Main.js (hunk 3 above)
Comment (sv-yoda) [inline src/components/Main.js]: Oh, that is from react-native-fcm library.

Comment (lyw0149) [reply src/components/Main.js:142]: Oh, that is from react-native-fcm library.

**Comment by ifndefdeadmau5** [comment]
After updating branch you can merge LGTM

[Approved with PullApprove](https://pullapprove.com/sv-bootcamp/bridgeme-client/pull-request/135/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=sv-bootcamp/bridgeme-client)

**Pull request merged**